src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/Multiplatform/StatusBar/StatusBar.uwp.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using Windows.Foundation.Metadata;
+using Windows.UI.ViewManagement;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.UWP;
+using ViewManagement = Windows.UI.ViewManagement;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.Multiplatform
+{
+	public static partial class StatusBar
+	{
+		static partial void PlatformSetColor(Color color)
+		{
+			var windowsColor = color.ToWindowsColor();
+			UpdateStatusBar(
+				sb => sb.BackgroundColor = windowsColor,
+				tb => tb.BackgroundColor = windowsColor);
+		}
+
+		static partial void PlatformSetStyle(StatusBarStyle style)
+		{
+			var foregroundColor = style switch
+			{
+				StatusBarStyle.LightContent => Color.White.ToWindowsColor(),
+				StatusBarStyle.DarkContent => Color.Black.ToWindowsColor(),
+				_ => Color.Default.ToWindowsColor(),
+			};
+
+			UpdateStatusBar(
+				sb => sb.ForegroundColor = foregroundColor,
+				tb => tb.ForegroundColor = foregroundColor);
+		}
+
+		static void UpdateStatusBar(Action<ViewManagement::StatusBar> updateStatusBar, Action<ApplicationViewTitleBar> updateTitleBar)
```

Comment:
Do these arguments need to be an `Action`? It's very simple to add this overhead.

Reply:
Yes. They need to be actions since we update `StatusBar` and `ApplicationViewTitleBar` in different ways from different methods.